### DISCUSSION TITLE
CGI Support

### DISCUSSION THREAD

**Discussion by E8y2FqZE** [sqlpage/SQLPage#249] [Ideas]
Is there currently any way to run SQLpage via CGI?

I have shared web hosting that would be complicated/impossible to start a long-running daemon process, but I have had great success dropping a binary or script in the public dir and tweaking `.htaccess`.

**Comment by lovasoa** [comment]
Hello and welcome to SQLPage!
  The software currently doesn't support the CGI protocol. Adding support wouldn't be straightforward since we are based on the actix-web library, which doesn't support CGI. 
  If you are looking for an easy hosting solution, you can subscribe to the wait-list for SQLPage cloud  here: https://sql.ophir.dev/your-first-sql-website/hosted.sql

 Alternatively, you can run SQLPage severless on AWS, which will probably be entirely free for you, unless you get a lot of traffic.
